models/models.go

```diff
@@ -97,14 +97,18 @@ func LoadConfigs() {
 
 func getEngine() (*xorm.Engine, error) {
 	cnnstr := ""
+	var Param string = "?"
+	if strings.Contains(DbCfg.Name,Param) {
+		Param="&"
+	}
```

Comment:
These above 4 lines can be removed by doing `?charset=utf8&parseTime=true%s` instead below which makes more sense.